packages/remix-server-runtime/__tests__/server-test.ts

```diff
@@ -880,7 +880,8 @@ describe("shared server runtime", () => {
       let result = await handler(request);
       expect(result.status).toBe(400);
       expect(testAction.mock.calls.length).toBe(1);
-      expect(rootLoader.mock.calls.length).toBe(1);
+      // Should not call root loader since it is the boundary route
+      expect(rootLoader.mock.calls.length).toBe(0);
```

Comment:
These unit tests were previously invalid - if we throw from the child and it gets handled by the root - we do not want to re-run the root loader